### ISSUE TITLE
store eth txn hash for blocks

### ISSUE THREAD

**Issue by jrhite** [omisego/elixir-omg#931]
The block view of the block explorer will display the eth txn hash of the txn that stored the block on the root chain.

See here: https://gallery.io/projects/MCHbtQVoQ2HCZSli4_VqE738/files/MCEJu8Y2hyDScSrQhNKu8JfDGlCFfA3vbv0

This info needs to be stored in the database. Schema and code changes required.

**Comment by jrhite** [comment]
Currently blocked on API split: https://github.com/omisego/elixir-omg/issues/1129

**Comment by unnawut** [comment]
Removing blocked tag since #1123 is merged and #1129 shouldn't be a blocker on this.

**Comment by unnawut** [comment]
Let's do the DB and API in one go so we don't have to maintain dangling code or DB fields that are not used. Closing in favor of #1232.
